### ISSUE TITLE
Implement new color styles for pattern library design

### ISSUE THREAD

**Issue by vz3** [18F/cg-design#11]
Typography, grid, palette

This is essential for pattern library

Need to outline documentation about typography, palette, etc.
Consider what lightest version of this would be
- Changing base styles/add to pattern library
- Build out a few patterns

Marco:
- Wants to take base styles and typographies and get them in sync with our prototypes
- Would also like to build some patterns for base styles but don’t want them to be a big burden
- Icons: 597
- When designers build UI components, name them (e.g. notifications) in the mockup so we are all using the same naming

Also discussed: 
- Create issue for workflow to add new patterns
- How to hand off?

Next step:
- Dan/Aaron/Marco/Victor sync, report back to team

Ref: https://github.com/18F/cg-design/issues/3

**Comment by suprenant** [comment]
A conversation for this is scheduled for Monday.

**Comment by thisisdano** [comment]
This is in progress.
See colors, type scale, and basic patterns at the project Invision: https://gsa.invisionapp.com/share/9T8GMJ8B8

We're maintaining the current Sketch file in the `cg-deign` repo: https://github.com/18F/cg-design/raw/master/Sketch/cg-atomic-bass.sketch

**Comment by suprenant** [comment]
Waiting on @thisisdano here to wrap up designs

**Comment by vz3** [comment]
This has been flagged as a blocker by @suprenant @msecret

**Comment by msecret** [comment]
Here's our current color palette. We should attempt to replace the primary/secondary values as they make sense with the new colors. We should also standardize what grays we're officially using. 
From there I want to set the contextual colors, `$color-error`, `$color-warning`, and hopefully reuse the primary and secondarys for those so, `$color-warning = $color-secondary;`.

``` scss
// color
$color-primary:              #056dd4;
$color-primary-darker:       #2c3e50;
$color-primary-darkest:      #1f2c38;

$color-primary-alt:          #3498db;
$color-primary-alt-dark:     darken($color-primary-alt, 40%);
$color-primary-alt-darkest:  darken($color-primary-alt, 70%);
$color-primary-alt-light:    #77b6ff;
$color-primary-alt-lightest: #d9f1ff;

$color-secondary:            #e74c3c;
$color-secondary-dark:       #b0392e;
$color-secondary-darkest:    darken($color-secondary, 75%);
$color-secondary-light:      #f3736e;
$color-secondary-lightest:   #deeeef; // DEEF!

$color-focus:                rgba(255, 221, 0, 0.55);
$color-visited:              #4c2c92;

$color-gold:                 #f8ba1c;

$color-finish:               #0daf02;
$color-warning:              #cb4f03;
$color-error:                #f16031;

$color-gray:                 #595959;
$color-gray-dark:            darken($color-gray, 20%);

$color-disclaimer-text:      #000000;
$color-disclaimer-link:      #0071bc;

$color-base: $color-gray;
$text-max-width:    660px;

// TODO what color is this? it should follow convention
$color-lightgray: #bababa;
$site-max-width:    1040px;

$color-background-light: #f3f4f8;
```

**Comment by thisisdano** [comment]
``` scss

// commenting out colors i don't expect to use

// color
$color-primary:              #056dd4;
//$color-primary-darker:       #2c3e50;
//$color-primary-darkest:      #1f2c38;

//$color-primary-alt:          #3498db;
//$color-primary-alt-dark:     darken($color-primary-alt, 40%);
//$color-primary-alt-darkest:  darken($color-primary-alt, 70%);
//$color-primary-alt-light:    #77b6ff;
//$color-primary-alt-lightest: #d9f1ff;

$color-secondary:            #CB4F03;
//$color-secondary-dark:       #b0392e;
//$color-secondary-darkest:    darken($color-secondary, 75%);
//$color-secondary-light:      #f3736e;
//$color-secondary-lightest:   #deeeef; // DEEF!

$color-focus:                rgba(255, 221, 0, 0.55); // slated for a change

// love to remove this one
$color-visited:              #4c2c92; 

$color-gold:                 #FEB000;

$color-finish:               #0daf00;
$color-warning:              $color-secondary;
$color-error:               $color-secondary;

// dunno how these are used
$color-gray:                 #595959;
$color-gray-dark:            darken($color-gray, 20%);

// here are my grays
$color-trueblack:               #000000;
$color-textblack:               #333333;
$color-darkgray:               #555555;
$color-mediumgray:               #8E8E8E;
$color-lightgray:               #D3D3D3;
$color-white:               #FFFFFF; // perhaps redundant


// don't know what these are
$color-disclaimer-text:      #000000;
$color-disclaimer-link:      #0071bc;

// assuming this is the primary text color
//$color-base: $color-gray;
$color-base: $color-textblack;

$text-max-width:    660px;

// TODO what color is this? it should follow convention
$color-lightgray: #D3D3D3;
$site-max-width:    1040px;

$color-background-light: #FAFAF8;
```

**Comment by thisisdano** [comment]
updating `$color-background-light` to `#F5F5F3`

**Comment by suprenant** [comment]
@msecret do you want a separate issue to track the integration work here that you're picking up, or should we just tweak the card's title and use this one?

**Comment by msecret** [comment]
@suprenant I think we can just use this one

**Comment by msecret** [comment]
Acceptance criteria:
- [ ] The color variables should be changed to new values.

**Comment by msecret** [comment]
Now that this is moving to design, I wanted to separate this issue out into different issues for colors, typography and grid. I'll keep this one for colors. I wanted to do this because each of these 3 things has a different set of time to complete and risks.

**Comment by suprenant** [comment]
Cool good call @msecret

**Comment by adborden** [comment]
Spot checked a few colors, looks good.
